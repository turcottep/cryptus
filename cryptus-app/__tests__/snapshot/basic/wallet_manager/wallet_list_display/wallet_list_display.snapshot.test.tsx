import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import WalletListDisplay from "../../../../../components/basic/wallet_manager/wallet_list_display/wallet_list_display";

describe("<WalletListDisplay />", () => {
  it("should display properly", async () => {
    const mockProps = {
      callback: null,
      currentAddress: "whateveranaddressissupposedtolooklike",
      addresses: ["address1", "address2"],
      added: true,
    };

    const tree = renderer.create(<WalletListDisplay {...mockProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
