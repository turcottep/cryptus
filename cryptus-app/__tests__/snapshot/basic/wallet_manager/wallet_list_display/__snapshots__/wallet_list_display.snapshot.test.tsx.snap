// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WalletListDisplay /> should display properly 1`] = `
<div>
  <ul>
    <div>
      <div>
        <div>
          Metamask
          <img
            src="/images/metamask_logo.png"
          />
        </div>
        <div>
          address1
        </div>
      </div>
      <div>
        <img
          onClick={[Function]}
          src="/icons/delete_icon.png"
        />
      </div>
    </div>
    <div>
      <div>
        <div>
          Metamask
          <img
            src="/images/metamask_logo.png"
          />
        </div>
        <div>
          address2
        </div>
      </div>
      <div>
        <img
          onClick={[Function]}
          src="/icons/delete_icon.png"
        />
      </div>
    </div>
  </ul>
</div>
`;
